sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -541,7 +541,10 @@ case class WeekOfYear(child: Expression) extends UnaryExpression with ImplicitCa
 
 // scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = "_FUNC_(timestamp, fmt) - Converts `timestamp` to a value of string in the format specified by the date format `fmt`.",
+  usage = """
+    _FUNC_(timestamp, fmt) - Converts `timestamp` to a value of string in the format specified by the date format `fmt`.
+    See `java.text.SimpleDateFormat` for valid date and time format patterns.
+  """,
```

Comment:
Hi, @johnhany97 .
Please open a PR to master first.

Reply:
If we bypass `master` branch, Apache Spark 3.0+ document will have the same issue.